runtime/README.md

```diff
@@ -270,6 +270,17 @@ this case, the new Runtime Firmware must:
 Caliptra Runtime Firmware SHALL implement a profile of the DICE Protection
 Environment (DPE) API.
 
+### PAUSER Privilege Levels
+
+Caliptra models PAUSER callers to its mailbox as having 1 of 2 privilege levels:
+
+* PL0 - High Privilege. Only 1 PAUSER in the SoC may be at PL0. The PL0 PAUSER
+  is denoted in the signed Caliptra firmware image. The PL0 PAUSER may call any
+  supported DPE commands.
```

Comment:
Only PL0 can use the CertifyKey command. Availability of CertifyKey output evidence signifies the PL0 status of the consumer

(or something like that)

Reply:
Done